packages/ui/src/accounts/modals/MoveFoundsModal/MoveFundsModal.tsx

```diff
@@ -34,39 +34,56 @@ const actionNameMapper = (lock?: LockType) => {
 export const MoveFundsModal = () => {
   const {
     hideModal,
-    modalData: { requiredStake, lock },
+    modalData: { requiredStake, lock, isFeeOriented },
```

Comment:
@WRadoslaw is the `isFeeOriented` flag used anywhere ?

Reply:
It's not, but it's should be. I'll cover usages in next commit.